modules/javafx.graphics/src/main/java/javafx/scene/layout/BorderImage.java

```diff
@@ -50,7 +50,7 @@
  * When applied to a Region with a defined shape, a BorderImage is ignored.
  * @since JavaFX 8.0
  */
-public class BorderImage implements Interpolatable<BorderImage> {
+public final class BorderImage implements Interpolatable<BorderImage> {
```

Comment:
same question about final fields.

except for:
- innerEdge/outerEdge (used by border) and
- image use in Region:799 (could be accessed via getImage())